resources/views/layouts/navigation.blade.php

```diff
@@ -5,7 +5,7 @@
 
 <nav>
     <div class="{{ $navClasses }}">
-        <div class="flex h-16 justify-between">
+        <div class="flex h-16 md:justify-end bg-pink-500/50 w-full justify-center md:bg-transparent ">
```

Comment:
This pretty good. I suggest to use `backdrop-blur` like in the **About** page to keep it consistence:

`flex h-16 md:justify-end bg-slate-900/50 backdrop-blur-sm w-full justify-center md:bg-transparent md:backdrop-blur-none`

Reply:
Thanks, @addeeandra Updated it.